Lib/pathlib/_abc.py

```diff
@@ -25,6 +35,88 @@ def _is_case_sensitive(parser):
     return parser.normcase('Aa') == 'Aa'
 
 
+def _get_copy_blocksize(infd):
```

Comment:
This whole section of additions feels very concrete for an `_abc` module. Do we maybe need/want a `pathlib._os` module?

Reply:
I agree and will move it to `pathlib._os`.

Just to note: it's used in `_abc` so that future user subclasses of `PathBase` will be able to benefit from the fast copy routines, e.g. if a user calls `FTPPath('blah').copy(Path('foo'))`, and `FTPPath.open().fileno()` returns a file descriptor, it will use the fast routines to perform the download even though it's not a local-to-local copy (at least from a high level user perspective).

Reply:
Yeah, I figured it was there for default implementations. I don't have a problem with `pathlib._abc` importing from `pathlib._os` (perhaps lazily?), provided we don't end up with `_os` importing from other parts of pathlib and creating cycles.